public/assets/css/haras_googlemap.css

```diff
@@ -0,0 +1,13 @@
+.haras-googlemap {
+    display: flex;
+    justify-content: center;
+    border: 1px solid var(--primary-color);
+    border-radius: 0.5em;
+    padding: 0.1em;
+}
+
+@media screen and (max-width: 800px) {
+    .haras-googlemap {
+        margin: 3em;
+    }
+}
```

Comment:
magic value + media query may be avoidable using bootstrap i think

Reply:
done